### PR TITLE
GitHub analysis script

### PR DESCRIPTION
**Pull request by Hixie**
This is the script I used for some of my recent research.

### CODE REVIEW

File: dev/githubanalysis/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+   permissions -> Members -> Read-only".
+
+   `members.txt`: List of all people who are expected to be members of
+   the flutter-hackers group, one to a line.
+
+   `exmembers.txt`: List of all people who were once members of the
+   flutter-hackers group, but are not currently members, one to a
+   line.
+
+Then, run: `dart run --enable-asserts bin/githubanalysis.dart`
```
Comment (keyonghan) [inline dev/githubanalysis/README.md:30]: Wondering how long it takes the script to finish?

**Review by keyonghan — APPROVED** [review]
LGTM considering these are non-prod scripts.

File: dev/githubanalysis/README.md (hunk 1 above)
Comment (Hixie) [reply dev/githubanalysis/README.md:30]: weeks :-)

Comment (Hixie) [reply dev/githubanalysis/README.md:30]: (that's why it caches so aggressively)

**Pull request merged**